100nocks166ryoryo/CSS/05/new.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+<meta charset="utf-8">
+<title></title>
+<meta name="keywords" content="">
+<meta name="description" content="">
+<link href="new.css" rel="stylesheet" media="all">
+</head>
+
+<body>
+        <ul>
+                <li>ピジョン</li>
+                <li>カイオーガ</li>
+                <li class="new">ゼラオラ</li>
+                <li>ガオガエン</li>
+              </ul>
```

Comment:
```suggestion
        </ul>
```
インデントは開始タグと揃えるようにしましょう